src/gatsby-node.ts

```diff
@@ -68,7 +85,7 @@ export async function sourceNodes(
         // construct url
         Url: `https://s3.${
           region ? `${region}.` : ""
-        }amazonaws.com/${Bucket}/${Key}`,
+          }amazonaws.com/${Bucket}/${Key}`,
```

Comment:
I think the file was re-formatted using different Prettier rules (v2 is seems?).

Would you mind formatting it using `yarn prettier src/gatsby-node.ts --write`? (it will use the prettier from the dependencies)

Reply:
Yes, sorry I didn't see this.

Reply:
No worries! Thanks 🙂